schema/deploy/tables/revision_type.sql

```diff
@@ -0,0 +1,43 @@
+-- Deploy cif:tables/revision_type to pg
+
+
+begin;
+
+create table cif.revision_type
+(
+  type varchar(1000) primary key
+);
+
+select cif_private.upsert_timestamp_columns('cif', 'revision_type');
+
+do
+$grant$
+begin
+
+-- Grant cif_internal permissions
+perform cif_private.grant_permissions('select', 'revision_type', 'cif_internal');
+perform cif_private.grant_permissions('insert', 'revision_type', 'cif_internal');
+perform cif_private.grant_permissions('update', 'revision_type', 'cif_internal');
+
+-- Grant cif_admin permissions
+perform cif_private.grant_permissions('select', 'revision_type', 'cif_admin');
+perform cif_private.grant_permissions('insert', 'revision_type', 'cif_admin');
+perform cif_private.grant_permissions('update', 'revision_type', 'cif_admin');
```

Comment:
We probably only need `select` permission here, at least while we don't have an admin/management view for this

```suggestion
-- Grant cif_internal permissions
perform cif_private.grant_permissions('select', 'revision_type', 'cif_internal');

-- Grant cif_admin permissions
perform cif_private.grant_permissions('select', 'revision_type', 'cif_admin');
```